src/components/AgendaItemCard.tsx

```diff
@@ -282,3 +294,271 @@ export function SearchResultAgendaItemCard({
     </Link>
   );
 }
+
+type Motion = {
+  committeeSlug: string;
+  motionType: string;
+  motionId: string;
+  voteDescription: string;
+  dateTime: string;
+  value: string;
+  tally: string;
+  resultKind: string;
+  committeeName: string;
+};
+
+type SearchPageAgendaItemCardProps = {
+  item: AgendaItem;
+  decisionBody: DecisionBody;
+  categories: string[];
+  motions: Motion[];
+};
+
+export function SearchPageAgendaItemCard({
+  item,
+  decisionBody,
+  categories,
+  motions,
+}: SearchPageAgendaItemCardProps) {
+  return (
+    <Link href={`/actions/item/${item.reference}`}>
+      <AgendaItemCard
+        item={item}
+        decisionBody={decisionBody}
+        className="transition-shadow hover:shadow-md"
+        Footer={() => (
+          <Button size="lg" className="w-full" variant="default">
+            Learn more
+          </Button>
+        )}
+      >
+        <div className="space-y-4">
+          {/* Category tags */}
+          <div className="flex flex-wrap gap-2">
+            {categories.map((category) => (
+              <Chip
+                key={category}
+                variant="secondary"
+                className="bg-neutral-100 dark:bg-neutral-800"
+              >
+                {category}
+              </Chip>
+            ))}
+          </div>
+
+          {/* Title and summary */}
+          <div>
+            <h3 className="text-lg font-semibold">{item.agendaItemTitle}</h3>
+            <div
+              className="mt-2 text-neutral-600 dark:text-neutral-300"
+              dangerouslySetInnerHTML={{ __html: item.agendaItemSummary }}
+            />
+          </div>
+
+          {/* Decision section if present */}
+          {item.agendaItemRecommendation && (
+            <div>
+              <h4 className="font-bold mb-2">The decision</h4>
+              <div
+                className="text-neutral-600 dark:text-neutral-300"
+                dangerouslySetInnerHTML={{
+                  __html: item.agendaItemRecommendation,
+                }}
+              />
+            </div>
+          )}
+
+          {/* Motions section using the councillor page style */}
+          <div className="mt-2">
```

Comment:
If pulling MotionsList into `src/components/MotionsList` to use it here should be fine